ops/_main.py

```diff
@@ -268,7 +274,9 @@ def run_any_legacy_hook(self):
         argv[0] = str(dispatch_path)
         logger.info('Running legacy %s.', self._dispatch_path)
         try:
-            subprocess.run(argv, check=True)
+            with tracer.start_as_current_span('ops._main::subprocess.run') as span:
+                span.set_attribute('argv', ' '.join(argv))
+                subprocess.run(argv, check=True)
```

Comment:
This is code we're removing pretty soon, right? If so, I don't think it's worth bothering to trace it.

Reply:
done, thx